front/src/applications/stdcm/components/StdcmWarningBox.tsx

```diff
@@ -32,6 +37,28 @@ const StdcmWarningBox = ({
   const hasMissingFields = (errorDetails?.missingFields?.length ?? 0) > 0;
   const hasRouteErrors = (errorDetails?.routeErrors?.length ?? 0) > 0;
 
+  const infraId = useSelector(getStdcmInfraID);
+  const { infra } = useInfraStatus({ infraId });
+  const { pathfinding, isPathFindingLoading: _ } = useStaticPathfinding(infra);
+  const isFailureWithIncompatibleConstraints =
+    pathfinding?.status === 'failure' &&
+    pathfinding.failed_status === 'pathfinding_not_found' &&
+    pathfinding.error_type === 'incompatible_constraints';
+
+  const numberErrorConstraintElectrical = isFailureWithIncompatibleConstraints
```

Comment:
To shorten a bit (same for the others)
```suggestion
  const electricalConstraintCount = isFailureWithIncompatibleConstraints
```

Reply:
Done